packages/angular-test-app/src/preview-examples/message-bar.ts

```diff
@@ -12,13 +12,15 @@ import { Component } from '@angular/core';
 @Component({
   selector: 'app-example',
   template: `
-    <ix-message-bar>Message text</ix-message-bar>
-    <ix-message-bar type="warning">Message text</ix-message-bar>
-    <ix-message-bar type="danger">
-      <div class="d-flex align-items-center justify-content-between">
-        Message text <ix-button>Action</ix-button>
-      </div>
-    </ix-message-bar>
+    <div style='margin: 0.5rem'>
+      <ix-message-bar style="margin-bottom: 1.25rem; display: block">Message text</ix-message-bar>
+      <ix-message-bar style="margin-bottom: 1.25rem; display: block" type="warning">Message text</ix-message-bar>
```

Comment:
Also for the other examples!

```suggestion
      <ix-message-bar style="margin-bottom: 0.5rem; display: block">Message text</ix-message-bar>
      <ix-message-bar style="margin-bottom: 0.5rem; display: block" type="warning">Message text</ix-message-bar>
```